### ISSUE TITLE
RFC: rewrite on top of Symfony console

### ISSUE THREAD

**Issue by robbieaverill** [silverstripe/sspak#54]
There are a number of issues logged on this repo which would be trivial to implement if sspak was written on top of the Symfony console component.

Benefits include:

- progress indication
- silent output options
- better handling of input arguments and options 
- a common framework for console commands
- extensibility

It would require a major version bump (probably) and a fair amount of work, but probably worthwhile in the long run.

**Comment by dhensby** [comment]
Sure - I've thought this too - but it's just is the effort worth the return?

**Comment by robbieaverill** [comment]
In the long run it probably is, although it might be hard to see right now

**Comment by dhensby** [comment]
Yep - and there's the problem, does anyone have the time to do it now?

**Comment by dhensby** [comment]
I've kind of started this lightly here: https://github.com/dhensby/sspak/tree/pulls/symfony-console
